quill-core/src/main/scala/io/getquill/norm/ApplyMap.scala

```diff
@@ -57,6 +57,12 @@ object ApplyMap {
         val er = BetaReduction(e, d -> c)
         Some(Map(SortBy(a, b, er, f), b, c))
 
+      // a.map(b => c).sortBy(d => e).distinct =>
```

Comment:
Having unit tests for the transformations makes it easier to maintain the code, could you add tests to `ApplyMapSpec`?

Reply:
Added to `ApplyMapSpec`